### DISCUSSION TITLE
Delete Webhooks should give information about what was deleted

### DISCUSSION THREAD

**Discussion by lriggle-strib** [directus/directus#2794] [Feature Requests]
## Feature Request
I would like to see the ID of the deleted item passed along in the body of Delete webhooks. That way any system receiving that webhook would know what was deleted.

## Background
I am currently a Directus Cloud customer and I am looking to use webhooks to trigger data changes in my organization's main API. The data from directus will be funneling into that application for now because of tight deadline constraints, but we are also looking at keeping the data in that API after our deadlines have passed as a kind of GraphQL Federation style setup (Apollo GraphQL has more information on this concept).

In testing, setting up the Create/Update webhooks work great. They tell me the ID of the item created/changed and also what fields are new/changed (which I was not expecting, that was a nice surprise). I can pass the specific collection type along as a query string parameter since the webhooks themselves are on a per-collection type basis, so I know exactly where my data needs to go and what ones need changes.

The Delete webhooks are the only ones that don't give me any information. I can set up a delete webhook, but the payload for it is completely empty. There is no identifying information about the content item passed along the delete webhook, making it difficult to tie it to a specific piece of data.

Here is what a delete webhook looks like from PostBin:
<img width="1206" alt="Screen Shot 2020-07-22 at 11 25 42 AM" src="https://user-images.githubusercontent.com/7051012/88205606-abade780-cc12-11ea-93e6-3dba74c0c8c5.png">

Here is what an update webhook looks like from PostBin:
<img width="1206" alt="Screen Shot 2020-07-22 at 11 29 00 AM" src="https://user-images.githubusercontent.com/7051012/88205658-bcf6f400-cc12-11ea-8ee1-fc7c98a017ec.png">

In the update webhook, there is an `id` field included as part of the body. The delete webhook has a completely empty body. I would like to see an `id` field included as part of the body of the delete webhook.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
